src/controllers/dashboard/scheduledtasks/scheduledtasks.js

```diff
@@ -132,8 +132,10 @@ function updateTaskButton(elem, state) {
 export default function(view) {
     function updateTasks(tasks) {
         for (const task of tasks) {
-            view.querySelector('#taskProgress' + task.Id).innerHTML = getTaskProgressHtml(task);
-            updateTaskButton(view.querySelector('#btnTask' + task.Id), task.State);
+            const taskProgress = view.querySelector('#taskProgress' + task.Id);
+            if (taskProgress) taskProgress.innerHTML = getTaskProgressHtml(task);
+            const btnTask = view.querySelector('#btnTask' + task.Id);
+            if (btnTask) updateTaskButton(btnTask, task.State);
```

Comment:
```suggestion
            const taskProgress = view.querySelector(`#taskProgress${task.Id}`);
            if (taskProgress) taskProgress.innerHTML = getTaskProgressHtml(task);

            const taskButton = view.querySelector(`#btnTask${task.Id}`);
            if (taskButton) updateTaskButton(taskButton, task.State);
```

Reply:
Fair enough, I originally just used the html id for variable name